near-contract-standards/src/event.rs

```diff
@@ -0,0 +1,264 @@
+use near_sdk::AccountId;
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+
+#[derive(Serialize, Debug)]
+#[serde(tag = "standard")]
+#[serde(rename_all = "snake_case")]
+pub enum NearEvent<'a> {
+    #[serde(borrow)]
```

Comment:
can remove all of these, only needed when using `Cow`

Reply:
It is because of the introduction of the lifetime.  For example, when removed:

```rust
cannot infer an appropriate lifetime for lifetime parameter `'de` due to conflicting requirements
expected `MapAccess<'_>`
   found `MapAccess<'de>`
expected `<NftMintData<'a> as Deserialize<'_>>`
   found `<NftMintData<'_> as Deserialize<'_>>`
   ```

Reply:
hm okay interesting, assumed the compiler could infer